server/manager/src/main/java/org/apache/accumulo/manager/upgrade/UpgradeCoordinator.java

```diff
@@ -142,11 +143,17 @@ public synchronized void upgradeZookeeper(ServerContext context,
         "Not currently in a suitable state to do zookeeper upgrade %s", status);
 
     try {
-      int cv = context.getServerDirs()
-          .getAccumuloPersistentVersion(context.getVolumeManager().getFirst());
-      ServerContext.ensureDataVersionCompatible(cv);
+      int cv = AccumuloDataVersion.getCurrentVersion(context);
       this.currentVersion = cv;
 
+      int oldestVersion = ROOT_TABLET_META_CHANGES;
+      if (cv < oldestVersion) {
+        String oldRelease = dataVersionToReleaseName(oldestVersion);
+        throw new UnsupportedOperationException("Upgrading from a version before " + oldRelease
```

Comment:
I think "less than" makes more sense here than "before", since "before" could mean chronologically, and we might release a bugfix for an earlier release line after the `.0` version we're expecting. But that bugfix's version would still be "less than" the `.0` version release.

```suggestion
        throw new UnsupportedOperationException("Upgrading from a version less than " + oldRelease
```